src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/NativeScrollContentPresenter.iOSmacOS.cs

```diff
@@ -150,7 +150,7 @@ public override void Layout()
 					UpdateDelayedTouches();
 #else
 					var size = AdjustContentSize(_content.Frame.Size + new CGSize(horizontalMargin, verticalMargin));
-					ContentView.Frame = new CGRect(new CGPoint(0, 0), size);
+					ContentView.Frame = new CGRect(_content.Frame.Location, size);
```

Comment:
I can see how this might account for the Left and Top margins - what about Right and Bottom?

Reply:
Right & bottom are the result of the size measurement. So it works :-)
@davidjohnoliver